resolver/auth_mutations.resolvers_test.go

```diff
@@ -58,139 +59,59 @@ const (
 	TestUsername               = "wednesday"
 	TestToken                  = "refreshToken"
 	ReqToken                   = "refresh_token"
-	ErrorMessage               = "an error '%s' was not expected when opening a stub database connection"
 )
 
-func TestLogin(t *testing.T) {
-	cases := prepareTestCases()
-
-	for _, tt := range cases {
-		t.Run(tt.name, func(t *testing.T) {
-			handleTestCases(t, tt)
-		})
-	}
-}
-
-type LoginArgs struct {
+type loginArgs struct {
 	UserName string
 	Password string
 }
 
-var req = LoginArgs{
-	UserName: testutls.MockEmail,
-	Password: OldPassword,
-}
-
-func prepareTestCases() []struct {
+type LoginType struct {
```

Comment:
this would be small case since it doesn't need to be used outside of the package